test/IDE/complete_call_arg.swift

```diff
@@ -486,28 +367,26 @@ class C4 {
 
 func firstArg(arg1 arg1: Int, arg2: Int) {}
 func testArg1Name1() {
-  firstArg(#^FIRST_ARG_NAME_1^#
+  firstArg(#^FIRST_ARG_NAME_1?check=FIRST_ARG_NAME_PATTERN^#
 }
 // FIRST_ARG_NAME_PATTERN: ['(']{#arg1: Int#}, {#arg2: Int#}[')']
 func testArg2Name1() {
-  firstArg(#^FIRST_ARG_NAME_2^#)
+  firstArg(#^FIRST_ARG_NAME_2?check=FIRST_ARG_NAME_PATTERN^#)
 }
 
 func testArg2Name3() {
-  firstArg(#^FIRST_ARG_NAME_3^#,
+  firstArg(#^FIRST_ARG_NAME_3?check=FIRST_ARG_NAME_PATTERN^#^#,
```

Comment:
Why did this one change away from FIRST_ARG_NAME_4 ?

Reply:
Ah good eye :) I forgot to add a comment about this.

`FIRST_ARG_NAME_4` and `FIRST_ARG_NAME_PATTERN` is basically the same.
 ```
 // FIRST_ARG_NAME_PATTERN: ['(']{#arg1: Int#}, {#arg2: Int#}[')']
...
 // FIRST_ARG_NAME_4: Decl[FreeFunction]/CurrModule: ['(']{#arg1: Int#}, {#arg2: Int#}[')'][#Void#];
 ```
Since I noticed I left extra `^#` at the end of this line, I'm going to change the check pattern of `FIRST_ARG_NAME_PATTERN` to the complete one.